admin/scripts/common/models/__init__.py

```diff
@@ -0,0 +1,105 @@
+from enum import Enum
+from typing import Any, Optional
+
+from pydantic import BaseModel, Field, field_validator, model_validator
+
+
+class Choice(str, Enum):
+    yes = "Yes"
+    not_planned = "Not planned"
+    not_yet = "Not yet"
+
+
+class Choice2(str, Enum):
+    yes = "Yes"
+    not_sure_yet = "Not sure yet"
+    no = "No"
+
+
+class ChoiceReminder(str, Enum):
+    no = "No"
+    yes_one_week = "Yes, in one week"
+    yes_one_month = "Yes, in one month"
+
+
+class ProjectRole(str, Enum):
+    principal_author_lead_engineer = "Principal author / lead engineer"
+    core_contributor_maintainer = "Core contributor / maintainer"
+    release_manager = "Release manager"
+    security = "Security"
+    infrastructure_devops = "Infrastructure/DevOps"
+    community_manager = "Community manager"
+    outreach_coordinator = "Outreach coordinator"
+    other = "Other"
+
+
+class UpdateFrequency(str, Enum):
+    cant_say = "Can't say"
+    more_once_per_day = "More than once per day"
+    more_once_per_week = "More than once per week"
+    more_once_per_month = "More than once per month"
+    more_once_per_year = "More than once per year"
+    less_once_per_year = "Less than once per year"
```

Comment:
If we can fix a unit for this (say `X times per Y`), we could perhaps validate and convert it directly in the field.

Reply:
If one wants to be really rigorous, these answer options can only be expressed as a dependent type: `don't know | interval x, where x < Y | more than Y`. It can be done dynamically, but then you need to check that invariant at instance construction, which is probably more work than it's worth. Let's keep it an enum for now.